<launch>
	<include file="/home/tommyz/audio_common/audio_capture/launch/capture.launch"/>

  <!-- arecord -l will show available input devices, use the car number as
      the first number and the subdevice number as the second in a string
      like hw:1,0 -->
  <arg name="device" default=""/>
  <arg name="channels" default="1"/>
  <arg name="sample_rate" default="16000"/>
  <arg name="format" default="mp3"/>
  <arg name="ns" default="audio"/>


    <node name="audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
      <param name="bitrate" value="128"/>
      <param name="device" value="$(arg device)"/>
      <param name="channels" value="$(arg channels)"/>
      <param name="sample_rate" value="$(arg sample_rate)"/>
      <param name="format" value="$(arg format)"/>
    </node>


  <node name="audio" pkg="audio_video" type="audio.py"/>

  <node name="audio_convert" pkg="bag2wav" type="bag2wav.cpp"/>

</launch>
